test/MUXControlsTestApp/TestInventory.cs

```diff
@@ -12,16 +12,27 @@ class TestInventory
         static TestInventory()
         {
             Tests = new List<TestDeclaration>();
-            foreach(Type type in typeof(TestInventory).GetTypeInfo().Assembly.GetTypes())
+            AxeTests = new List<TestDeclaration>();
+            foreach (Type type in typeof(TestInventory).GetTypeInfo().Assembly.GetTypes())
             {
-                var attribute = type.GetTypeInfo().GetCustomAttribute<TopLevelTestPageAttribute>();
-                if(attribute != null)
+                var attributeTest = type.GetTypeInfo().GetCustomAttribute<TopLevelTestPageAttribute>();
```

Comment:
I don't quite understand the motivation for this change. Why are "axe tests" being treated as fundamentally different from the rest of the tests? Surely "axe validation" is just a particular technique that can be used. Why the need to add this extra complexity to the test app?

Reply:
I wanted to separate AXE testing out as the regular test pages definitely throw AXE issues so are no use for that. On the other hand, AXE testing pages don't offer much beside being tested for AXE testing. If you want, I can add them to the regular test cases, however over time, that will bloat the list of test pages list.

Reply:
Ok, I see now. In some cases you added the attribute to an existing page (AutoSuggestBox) and in others added a new page.
You're right that we would not want to add a ton of new top-level test pages. 

Can you add a comment to TestInventory.cs explaining the motivation? Also mention that in some cases the "axe page" is just an existing test page for a Control, and in other cases it is a specially created page for this purpose.

Reply:
> Ok, I see now. In some cases you added the attribute to an existing page (AutoSuggestBox) and in others added a new page.

Yes, exactly. The AutoSuggestBox test page has nothing beside AutoSuggestBox controls and as such is also suitable for AXE testing.

> Can you add a comment to TestInventory.cs explaining the motivation? Also mention that in some cases the "axe page" is just an existing test page for a Control, and in other cases it is a specially created page for this purpose.
Sure thing, Added a comment for that now.